utils/diffing.mli

```diff
@@ -14,7 +14,7 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** {0 Parametric diffing}
+(** {1 Parametric diffing}
```

Comment:
That header was intended to be the module title, isn't that the `0` level in odoc?

Reply:
Odoc doesn't allow to write a heading of level 0:
```
File "utils/diffing.mli", line 17, characters 4-26:
Warning: '{0': heading level should be lower than top heading level '0'.
```

Do you think it should allow that if it's at the top of a module (before any code and doc) ?
Odoc currently generate an equivalent of a level 0 heading, "Module Diffing".

Reply:
Ah right, this is a difference between `mld` and `mli`, isn't it? It might be better to enforce some homogeneity.

In this case, the simplest correction is probably to transform this heading into a preamble rather than lower its level (or remove it altogether).

Reply:
`mld` requires a level-0 heading, so it's a bit inconsistent but I'm not sure it's a good idea to allow redefining the title for modules. I've opened an issue in Odoc just in case, https://github.com/ocaml/odoc/issues/815.

Reply:
Ah sorry, I meant the reverse: that it was indeed better to enforce some homogeneity on titles for modules.